components/project-card/meta/get-involved.jsx

```diff
@@ -20,40 +21,49 @@ class GetInvolved extends React.Component {
     let props = this.props;
     let getInvolvedText = props.getInvolved ? props.getInvolved : null;
     let getInvolvedLink = props.getInvolvedUrl ? (
-      <a
-        href={props.getInvolvedUrl}
-        target="_blank"
-        onClick={() => this.handleGetInvolvedLinkClick()}
-      >
-        Get Involved
-      </a>
+      <p className="mt-2">
+        <strong>
+          <a
+          href={props.getInvolvedUrl}
+          target="_blank"
+          onClick={() => this.handleGetInvolvedLinkClick()}
+          >
+            {props.getInvolvedUrl}
+          </a>
+        </strong>
+      </p>
     ) : null;
     if (!getInvolvedText && !getInvolvedLink) return <p>{DEFAULT_TEXT}</p>;
 
     return (
-      <p>
-        {getInvolvedText} {getInvolvedLink}
-      </p>
+      <div>
+        <p>{getInvolvedText}</p>
+        {getInvolvedLink}
+      </div>
     );
   }
 
   renderHelpLabels() {
     if (!this.props.helpTypes) return null;
 
-    return this.props.helpTypes.map(helpType => {
+    let helpLabel = this.props.helpTypes.map(helpType => {
       return (
         <Link
           to={`/projects?helpType=${encodeURIComponent(helpType)}`}
-          className="btn btn-xs btn-tag"
+          className={`btn btn-tag ${this.props.helpTypesOnModeration}`}
```

Comment:
Let's change the way we use this new prop.

In `project-card-simple.jsx`:

Change

```jsx
<GetInvolved
  helpTypesOnModeration="btn-xs"
  ...
/>
```

to

```jsx
<GetInvolved
  helpTypesOnModeration="btn-xs"
  onModerationMode={true}
  ...
/>
```
And in `renderHelpLabels() {}` in `components/project-card/meta/get-involved.jsx`

```jsx
let classes = classNames(`btn btn-tag`, {
  "btn-xs": this.props.onModerationMode
});
```